Alexey/Text/regex_query_tool/main/regex_query_tool.py

```diff
@@ -0,0 +1,27 @@
+def main():
+    pattern = str(input("Enter a pattern: "))
+    print(reg_expression(pattern))
+
+
+def data_loader():
+    names = []
+    with open('data.txt', 'r', encoding="utf-8") as f:
+        reader = f.readlines()
+        for row in reader:
+            names.append(row)
+    return names
+
+
+def reg_expression(pattern):
+    names = data_loader()
+    matches = []
+    for name in names:
+        if name.startswith(pattern):
```

Comment:
It doesn't look like regex function
Try to add test 
Pattern: '\d{2}-\d{2}-\d{4}'
Text: ABC 67-8945 12-01-2009
Should find: '12-01-2009'